scripts/scr_add_man/scr_add_man.gml

```diff
@@ -1,47 +1,48 @@
-function scr_add_man(argument0, argument1, argument2, argument3, argument4, argument5, argument6, argument7, argument8, argument9, argument10) {
 
-	// argument0 = role
-	// argument1 = company
-	// argument2 = armourset
-	// argument3 = weaponset
-	// argument4 = gear
-	// argument5 = exp
-	// argument6 = name
-	// argument7 = chaos
-	// argument8 = gear provided?
-	// argument9 = home, shipXY, or default
-	// argument10 = mobility
+function scr_add_man(man_role, target_company, choice_armour, choice_weapons, choice_gear, spawn_exp, spawn_name, corruption, other_gear, home_spot, mobility_items) {
 
-	// That should be sufficient to add stuff in a highly modifiable fashion
+	// man_role = role
+	// target_company = company
+	// choice_armour = armorset
+	// choice_weapons = weaponset
+	// choice_gear = gear
+	// spawn_exp = exp
+	// spawn_name = name
+	// corruption = chaos
+	// other_gear = gear provided?
+	// home_spot = home, shipXY, or default
+	// mobility_items = mobility
```

Comment:
these comments are superflous, except perhaps home_spot and other_gear.

Reply:
replaced with up to date and relevant comment